setup.py

```diff
@@ -64,11 +64,9 @@ def find_version(*file_paths):
         for key, value in extras_require.items():
             if key.startswith(':') and pkg_resources.evaluate_marker(key[1:]):
                 install_requires.extend(value)
-except Exception:
-    logging.getLogger(__name__).exception(
-        'Failed to compute platform dependencies. All dependencies will be '
-        'installed as a result.'
-    )
+except Exception as e:
+    print("Failed to compute platform dependencies: {}. ".format(e) +
+          "All dependencies will be installed as a result.", file=sys.stderr)
```

Comment:
I was hitting this exception, but I wasn't seeing the log message before logging isn't initialized yet. I replaced it with print and included the error so it's easier to debug.